processor/src/operations/io_ops.rs

```diff
@@ -225,24 +225,24 @@ where
     // ADVICE INPUTS
     // --------------------------------------------------------------------------------------------
 
-    /// Removes the next element from the advice tape and pushes it onto the stack.
+    /// Removes the next element from the advice stack and pushes it onto the stack.
```

Comment:
Let's change this to: "Pops an element from the advice stack and pushes it onto the operand stack."

Reply:
Done!